test-e2e/cip/e2e.go

```diff
@@ -140,114 +140,27 @@ func testSetup(repoRoot string, t E2ETest) error {
 		return err
 	}
 
-	pushRepo := getBazelOption(repoRoot, "STABLE_TEST_STAGING_IMG_REPOSITORY")
+	goldenPush := fmt.Sprintf("%s/test-e2e/cip/push-golden.sh", repoRoot)
 
-	if pushRepo == "" {
-		return fmt.Errorf(
-			"could not dereference STABLE_TEST_STAGING_IMG_REPOSITORY")
-	}
-
-	cmds := [][]string{
-		// In order to create a manifest list, images must be pushed to a
-		// repository first.
-		{
-			"bazel",
-			"run",
-			fmt.Sprintf(
-				"--workspace_status_command=%s/workspace_status.sh",
-				repoRoot),
-			"//test-e2e/cip:push-golden",
-		},
-		{
-			"docker",
-			"manifest",
-			"create",
-			fmt.Sprintf("%s/golden-foo/foo:1.0", pushRepo),
-			fmt.Sprintf("%s/golden-foo/foo:1.0-linux_amd64", pushRepo),
-			fmt.Sprintf("%s/golden-foo/foo:1.0-linux_s390x", pushRepo),
-		},
-		// Fixup the s390x image because it's set to amd64 by default (there is
-		// no way to specify architecture from within bazel yet when creating
-		// images).
-		{
-			"docker",
-			"manifest",
-			"annotate",
-			"--arch=s390x",
-			fmt.Sprintf("%s/golden-foo/foo:1.0", pushRepo),
-			fmt.Sprintf("%s/golden-foo/foo:1.0-linux_s390x", pushRepo),
-		},
-		{
-			"docker",
-			"manifest",
-			"inspect",
-			fmt.Sprintf("%s/golden-foo/foo:1.0", pushRepo),
-		},
-		// Finally, push the manifest list. It is just metadata around existing
-		// images in a repository.
-		{
-			"docker",
-			"manifest",
-			"push",
-			"--purge",
-			fmt.Sprintf("%s/golden-foo/foo:1.0", pushRepo),
-		},
-		// Remove tag for tagless image.
-		{
-			"gcloud",
-			"container",
-			"images",
-			"untag",
-			"--quiet",
-			fmt.Sprintf("%s/golden-foo/foo:NOTAG-0", pushRepo),
-		},
-	}
-
-	for _, cmdparts := range cmds {
-		c := cmdparts[0]
-		args := cmdparts[1:]
-
-		cmd := command.NewWithWorkDir(
-			repoRoot,
-			c,
-			args...,
-		)
-
-		fmt.Printf("executing %s\n", cmd.String())
-
-		std, err := cmd.RunSuccessOutput()
-		if err != nil {
-			return err
-		}
-
-		fmt.Println(std.Output())
-		fmt.Println(std.Error())
-	}
+	cmd := command.NewWithWorkDir(
+		repoRoot,
+		"/bin/bash",
+		goldenPush,
+		repoRoot,
+	)
 
-	return nil
-}
+	fmt.Printf("executing %s\n", cmd.String())
```

Comment:
I think it's fine to leave the STDOUT and STDERR as-is without klog, because it's nice to see the output of the actual command that was executed as-is (without wrapping it inside klog).